src/imf.jl

```diff
@@ -0,0 +1,78 @@
+# This file is a part of AstroLib.jl. License is MIT "Expat".
+
+function _imf{T<:AbstractFloat}(mass::AbstractVector{T}, expon::AbstractVector{T},
+                                mass_range::AbstractVector{T})
+    ne_comp = length(expon)
+    if length(mass_range) != ne_comp + 1
+        println("Length of array mass_range is not one more than that of expon")
+        return zeros(mass)
+    end
+    integ = ones(T, ne_comp)
+    joint = ones(T, ne_comp)
+    norm = ones(T, ne_comp)
+    for i = 1:ne_comp
+        if expon[i] != -1
+            integ[i] = (mass_range[i+1]^(1 + expon[i]) - mass_range[i]^(1 + expon[i]))/
+                       (1 + expon[i])
+        else
+            integ[i] = log(mass_range[i+1]/mass_range[i])
+        end
+        if i != 1
+            joint[i] = joint[i-1]*(mass_range[i]^(expon[i-1] - expon[i]))
+        end
+    end
+    norm = (1/sum_kbn(integ.*joint))*joint
```

Comment:
`sum_kbn` should be indeed more accurate, for example in the case of elements spanning _several_ order of magnitudes, but if this isn't the case we shouldn't worry too much ;-)

Anyway, I just realized that `sum(integ .* joint)` is simply the scalar product between the two vectors, so `dot(integ, joint)` is better.  Sorry for not having noticed before.